_pkgdown.yml

```diff
@@ -30,122 +30,162 @@ navbar:
     href: reference/index.html
   - text: Articles
     href: articles/index.html
+  - text: Examples
+    href: articles/index.html
   - text: News
     href: news/index.html
 
 reference:
-- title: High-level Functions
+- title: "High-level Functions"
```

Comment:
You should prefer sentence case instead of title case for these "titles" which are actually "headings".

Reply:
Nice suggestion! Can we keep the titles here, since the new pkgdown layout only serves as an initialization for Keaven's review?